index.html

```diff
@@ -4,9 +4,39 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <link rel="stylesheet" href="/style.css">
```

Comment:
- [ ] please remove the forward-slash (/) from the `href` here because the style sheet is not loaded because of this.